.pipelines/.vsts-vhd-scanning.yaml

```diff
@@ -0,0 +1 @@
+# empty file WIP
```

Comment:
what will the pipeline be used for?

Reply:
In addition to scanning during vhd builds, infra? (someone?) wants trivy scans for like n-1 images in production. So this pipeline will pretty much do the same process of scanning vhd vms, just for more images. 

so if we have a vhd and after a couple weeks a new CVE is found, we have like a daily scan that will update an infra table that they're making to track cve's.